src/themes/custom/app/submission/sections/upload/file/section-upload-file.component.ts

```diff
@@ -0,0 +1,89 @@
+import {
+    Component, Input, ViewChild
+} from '@angular/core';
+
+import {
+    SubmissionFormsModel
+} from 'src/app/core/config/models/config-submission-forms.model';
+import {
+    SubmissionSectionUploadFileEditComponent
+} from 'src/app/submission/sections/upload/file/edit/section-upload-file-edit.component';
+import {
+    SubmissionSectionUploadFileComponent as BaseComponent
+} from 'src/app/submission/sections/upload/file/section-upload-file.component';
+
+/**
+ * This component represents a single bitstream contained in the submission
+ */
+@Component({
+  selector: 'ds-submission-upload-section-file',
+  // styleUrls: ['./section-upload-file.component.scss'],
+  styleUrls: ['../../../../../../../app/submission/sections/upload/file/section-upload-file.component.scss'],
+  // templateUrl: './section-upload-file.component.html'
+  templateUrl: '../../../../../../../app/submission/sections/upload/file/section-upload-file.component.html'
+})
+export class SubmissionSectionUploadFileComponent
+    extends BaseComponent {
+
+  /**
+   * The list of available access condition
+   * @type {Array}
+   */
+  @Input() availableAccessConditionOptions: any[];
```

Comment:
I don't believe you need to repeat these Inputs here?  I think all you need is an empty component which `extends BaseComponent`.  At least that is what other components in `themes/custom` all look like.


So, I think this can be simplified to just include your `@Component` tag above, and then have:
```
export class SubmissionSectionUploadFileComponent
    extends BaseComponent {}
```

Reply:
I think you're correct here.  I think I was remembering having to do that in the Themed* version.  Mine is the only component in `custom` which has those.